src/ucs/sys/sys.c

```diff
@@ -933,14 +932,25 @@ double ucs_get_cpuinfo_clock_freq(const char *header, double scale)
     fclose(f);
 
     if (warn) {
-        level = ucs_global_opts.warn_inv_tsc ? UCS_LOG_LEVEL_WARN :
-                                               UCS_LOG_LEVEL_DEBUG;
-        ucs_log(level,
-                "Conflicting CPU frequencies detected, using: %.2f", value);
+        ucs_warn_inv_tsc("Conflicting CPU frequencies detected, using: %.2f",
+                         value);
     }
+
     return value * scale;
 }
 
+void ucs_warn_inv_tsc(const char *msg, ...)
```

Comment:
can we make it macro instead on inline function? would be simpler..

Reply:
it's not inline (at least not requested and has declaration).
ok, will create macro instead